apps/meteor/app/push/server/fcm.ts

```diff
@@ -54,6 +54,8 @@ type FCMError = {
 	};
 };
 
+type FetchOptions = NonNullable<Parameters<typeof fetch>[1]>;
```

Comment:
IF this is the same type of FetchOptions the package receives, I feel we can export this one from the package itself and import here (as it may be used later)

Reply:
![image](https://github.com/RocketChat/Rocket.Chat/assets/76006232/a8b653bd-4fee-4940-b4b2-f3899023d7df)

Reply:
Done